c7n/resources/vpc.py

```diff
@@ -3272,3 +3272,140 @@ def process(self, resources, event=None):
                 results.append(resource)
 
         return results
+
+
+@Vpc.filter_registry.register('resolver-rules-associated')
```

Comment:
For consistency/brevity, we might want to use a singular/shorter name like `resolver-rule` here